pkg/openshift/consoleplugin/nginx_configmap.go

```diff
@@ -34,6 +34,9 @@ http {
 		location / {
 			root                /usr/share/nginx/html;
 		}
+		location /config.js {
```

Comment:
out of curiosity. What is this doing?

Reply:
Basically:

- there's a new custom entrypoint script in the plugin image
- that script injects the two env vars into config.js
- the topology view in plugin loads this config.js, with the correct pointers to the ns/name of the topology configmap

Reply:
Thanks for the explanation.